### ISSUE TITLE
Accessibility issue where information not being conveyed

### ISSUE THREAD

**Issue by tracytang11** [SAP/fundamental-ngx#9916]
#### Is this a bug, enhancement, or feature request?

Accessibility Issue.

##### Observed Behavior:
On accessing the list items in the list box ‘Order status’, the information that the accessed list item is in selected/not selected state is not conveyed to the end user.

##### Screenshot:
<img width="452" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/134352250/7b981b11-a607-4871-99e5-ff938a49a121">

##### Expected Behavior: 
All UI elements should be presented correctly using assistive technologies. Here the user should be conveyed the correct information about the UI element accessed. Here the information that the accessed list item is in selected/not selected state should be conveyed to the end user.

##### How to reproduce: 
1. You can reproduce this problem when you use a multi-input. 
2. Open the accessibility testing tool
3. Selected one or two items. 

#### Is there anything else we should know?
We hope you can increase the specified attribute of aria to solve this issue.
Thanks!

**Comment by droshev** [comment]
@tracytang11 Can you test this in the latest version and also can you provide the log history from JAWS?

**Comment by droshev** [comment]
@tracytang11 Any update?

**Comment by droshev** [comment]
no activities => closing the issue
